com.vrcfury.vrcfury/Editor/VF/Hooks/Av3EmuAnimatorFixHook.cs

```diff
@@ -75,6 +79,21 @@ internal class Av3EmuAnimatorFixHook : IVRCSDKPreprocessAvatarCallback {
                     restartField.SetValue(emulator, true);
                 }
 
+                if (emulators != null && emulators.Length >= 1)
+                {
+                    foreach (var contactReceiver in avatar.GetComponentsInChildren<VRCContactReceiver>()) contactReceiver.paramAccess = null;
+                    foreach (var physBone in avatar.GetComponentsInChildren<VRCPhysBone>())

```

Comment:
BTW, the reason this didn't fix SPS is because you didn't include inactive components here.